feat: live painting comfy

pixelass opened the pull request:
## Motivation

<!-- List motivation and changes here -->

## Issues closed

<!-- List closed issues here -->

pixelass committed:
feat: add live painting with comfyui

pixelass committed:
feat: live-painting interaction

pixelass committed:
feat: live-painting interaction

pixelass committed:
refactor: remove unused functions

pixelass committed:
feat: throttle calls

pixelass committed:
fix: prevent queue overload

queue cannot exceed length 2

TimPietrusky commented:
## Feedback

* There is a huge delay & glitch when changing the prompt
* The start-button provides no state, so it's not clear why it should be clicked or what has changed when it was clicked

pixelass committed:
fix: performance and registration

- also adjusts styles

pixelass committed:
feat: higher FPS

pixelass committed:
feat: improve painting apps

- save for live painting
- reuse shared components
- better styles
- prepares for extraction to captn
- prepare for various image sizes

pixelass committed:
feat: adjust start button on live painting

TimPietrusky commented:
:tada: This PR is included in version 1.0.0-alpha.71 :tada:

The release is available on [GitHub release](https://github.com/blib-la/captain/releases/tag/v1.0.0-alpha.71)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
